### DISCUSSION TITLE
Video not Autoplaying when inside cypress test runner

### DISCUSSION THREAD

**Discussion by only4arunkumar** [cypress-io/cypress#16787] [Questions and Help]
As part of my testing i am supposed ensure that a video with allow autoplay flag in the iframe and queryString param (specific to our player)  starts to play without any user gesture.

It used to work fine , but with latest cypress our test starts to fail as  video are not autoplayed. 
The video is embedded in an Iframe. 

I am including a sample repo to replicate the issue.  When  accessing the url manually from the browser plays the video without any issue.
https://github.com/only4arunkumar/CypressAutoPlay

**Comment by jennifer-shehane** [comment]
Which Cypress version was working and which Cypress version is now not working?

**Reply by only4arunkumar** [reply]
Hi @jennifer-shehane  , thanks for your time.   I am currently using
"@testing-library/cypress": "^7.0.6",   
 "cypress": "^7.4.0",   
  "cypress-iframe": "^1.0.1"
  
  It used to work on 6.8.0 Cypress version

**Comment by only4arunkumar** [comment]
Hi @jennifer-shehane  
Thanks for your time. i have raised an issue for the same.  I have also checked for some community feedback  giiter community channel
https://github.com/cypress-io/cypress/issues/16824
